libs/lambda-edge-openid-auth/src/lib/config.ts

```diff
@@ -54,19 +55,22 @@ export async function getConfig(
             publicUrl,
             domain: request.headers.host[0].value,
             redirectUri: `${publicUrl}${callbackPath}`,
+            scope: 'openid offline_access email profile',
```

Comment:
```suggestion
            scope: rawConfig.scope, 
```

you'll also need to update the type of rawConfig to something like this:

```typescript
export interface RawConfig {
    unauthenticatedPaths: string[]
    idps: RawIdp[]
    scope?: string
}
```